src/darwin/Framework/CHIP/CHIPDeviceController.mm

```diff
@@ -218,7 +223,7 @@ - (BOOL)startup:(_Nullable id<CHIPPersistentStorageDelegate>)storageDelegate
         chip::MutableByteSpan icac;
 
         errorCode = _operationalCredentialsDelegate->GenerateNOCChainAfterValidation(
-            _localDeviceId, 0, ephemeralKey.Pubkey(), rcac, icac, noc);
+            _localDeviceId, 0, _localDeviceCATs, ephemeralKey.Pubkey(), rcac, icac, noc);
```

Comment:
Again, this CATs stuff should be coming from the consumer of the framework, not from this code....

Can this code just pass "no cats" to `GenerateNOCChainAfterValidation` and not add these bits that I'm pretty sure we don't actually want inside the framework?

Reply:
Via whatever API the framework will define, but "read some random storage key" is not necessarily that API.

Please just leave the Darwin code alone and file an issue about adding CAT support there as needed.

Reply:
Updated to assign `kUndefinedCATs`. Will create separate ticket to define proper APIs here.